extensions/integration-tests/src/notebook.test.ts

```diff
@@ -36,6 +36,26 @@ if (context.RunTest) {
 			assert(result === '2', `Expected: 2, Acutal: '${result}'`);
 		});
 
+		test('Clear all outputs - Python3 notebook ', async function () {
+			let notebook = await openNotebook(pySparkNotebookContent, pythonKernelMetadata);
+			//Check if at least one cell with output
+			let cellWithOutputs = notebook.document.cells.find(cell => cell.contents && cell.contents.outputs && cell.contents.outputs.length > 0);
```

Comment:
there should be an assert here to ensure the cellWithOutputs isn't undefined. also please add some more logging (take a look at what @YurongHe did earlier today)

Reply:
Will add. Will check with Yurong, however, I wonder what more logging is needed here as it's 4 lines of code and we have two logging statements, one test and the other one is assert.